Gruntfile.js

```diff
@@ -181,5 +169,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-browserify');
 
   grunt.registerTask('default', ['exec:bower_update','clean','copy:fonts','sass:dev','browserify:dev','exec:build_sphinx','connect','open','watch']);
-  grunt.registerTask('build', ['exec:bower_update','clean','copy:fonts','sass:build','browserify:build','uglify','exec:build_sphinx']);
+  grunt.registerTask('build', ['exec:bower_update','clean','copy:fonts','sass:build','browserify:build','uglify']);
```

Comment:
This removed the actual building of the docs. Was that intentional?

Reply:
Yes, the way I see this is that `grunt build` should just build the theme related files.

Reply:
I'm good with it. I just want to be sure.